src/views/RegisterPage/index.js

```diff
@@ -1,10 +1,18 @@
+import { useState } from 'react';
+import { CognitoUserPool } from 'amazon-cognito-identity-js';
 import React, { Component } from "react";
 import Form from "react-bootstrap/Form";
 import Button from "react-bootstrap/Button";
 import { Link, Route } from "react-router-dom";
 import LoginPage from "../LoginPage";
 
 
+const poolData = {
```

Comment:
let's move this into the .env file, take a look at how to do it [here](https://create-react-app.dev/docs/adding-custom-environment-variables/#adding-development-environment-variables-in-env). Essentially in .env.example put the `COGNITO_USER_POOL_ID=` and `COGNITO_CLIENT_ID=` (and commit that file), then in .env (which would should NOT commit) actually fill those out. when you run `npm start` it should automatically inject the environment variables. Then to access them just do `process.env.COGNITO_USER_POOL_ID`